### ISSUE TITLE
Provide mechanism for defining parser text

### ISSUE THREAD

**Issue by steviecoaster** [michaeltlombardi/tutti#1]
It is up to the tool author to define the help text for their tool. Because of this we will need to provide a way for users of Tutti to provide their own regex to the parser for different aspects of the native command.

For example, the tool may use `USAGE`, or `USAGE:`, or `Usage`. We _could_ write a greedy regex that will _probably_ work for many use-cases, however defaulting to this greedy regex and making it a parameter to be overridden will un-block the end user should they run into unexpected behavior

**Comment by michaeltlombardi** [comment]
Yeah, I think even for our own direct consumption this will be important - if the `gh` cli updates their help text, we don't want to be caught out trying to update hardcoded values all over the place.
